@xen-orchestra/rest-api/src/rest-api/rest-api.type.mts

```diff
@@ -5,6 +5,7 @@ import type {
   XenApiHostWrapped,
   XenApiMessage,
   XenApiNetworkWrapped,
+  XenApiPci,
```

Comment:
I suppose it is more linked to xen-api but I wonder why some objects are wrapped in WrapperXenApi and some are not.

Reply:
PR: #8692